src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -36,37 +36,43 @@ class DynatraceMetricsExporter(MetricsExporter):
     -------
     export(metric_records: Sequence[MetricRecord])
     """
-    __logger = logging.Logger(__name__)
 
     def __init__(
         self,
         endpoint_url: Optional[str] = None,
         api_token: Optional[str] = None,
         prefix: Optional[str] = None,
-        tags: Optional[Mapping[str, str]] = None,
+        default_dimensions: Optional[Mapping[str, str]] = None,
         export_oneagent_metadata: Optional[bool] = False,
     ):
+        self.__logger = logging.getLogger(__name__)
+
         if endpoint_url:
             self._endpoint_url = endpoint_url
         else:
             self.__logger.info("No Dynatrace endpoint specified, exporting "
                                "to default local OneAgent ingest endpoint.")
             self._endpoint_url = "http://localhost:14499/metrics/ingest"
 
-        all_tags = tags or {}
+        all_dimensions = default_dimensions or {}
 
         if export_oneagent_metadata:
             enricher = OneAgentMetadataEnricher()
-            enricher.add_oneagent_metadata_to_tags(all_tags)
+            enricher.add_oneagent_metadata_to_dimensions(all_dimensions)
 
-        self._serializer = DynatraceMetricsSerializer(prefix, all_tags)
+        self._serializer = DynatraceMetricsSerializer(prefix, all_dimensions)
         self._session = requests.Session()
         self._headers = {
             "Accept": "*/*; q=0",
             "Content-Type": "text/plain; charset=utf-8",
         }
         if api_token:
-            self._headers["Authorization"] = "Api-Token " + api_token
+            if not endpoint_url:
+                self.__logger.warning("Just API token but no endpoint passed. "
```

Comment:
```suggestion
                self.__logger.info("Just API token but no endpoint passed. "
```

Reply:
We should have clear guidelines for logging levels. IMO this would be a warn

Reply:
Ah I confused the context here - passing a token while using the local endpoint indicates a misconception we should warn users about indeed, so warning is fine here. Thanks for double-checking @dyladan!